app/services/msg/upcoming.rb

```diff
@@ -4,25 +4,27 @@ def respond
     events = Event.upcoming.limit(10)
     response = ""
     events.each do |event|
```

Comment:
What if instead we changed this line to `@person.department.events` ? Then we don't need the conditional later ?

Reply:
nice .. that works better